src/sentry/static/sentry/app/views/releases/detail/releaseActions.tsx

```diff
@@ -1,65 +1,144 @@
+/* eslint-disable react/prop-types */
```

Comment:
should we really do that? I noticed that we usually define the `propTypes` when it's necessary.